### ISSUE TITLE
Bug with Openfield demo frames labelling on Windows

### ISSUE THREAD

**Issue by vigji** [DeepLabCut/DeepLabCut#325]
Running DeepLabCut 2.0 in an anaconda3 env created from the windowsCPU config file.

When running the Demo_labeledexample_Openfield.ipynb demo, I get an error in generating the annotated frames from the annotation files and the raw frames. 

The problem is that line 292 of trainingsetmanipulation.py generate the labelled frame filename splitting the image name with the os.sep. However, the filename string read from CollectedData_Pranav.h5 seem to have the unix separator. Therefore, it does not split the filename and it never creates the labelled frames since it tries to access a non-existing folder.

**Comment by MMathisLab** [comment]
Can you provide the full error message please? 

We do not have this behavior on our end; as, the data is already formatted into a training set when you run ``load_demo_data``

**Comment by vigji** [comment]
Even after I run load_demo_data, when I run `deeplabcut.check_labels(path_config_file)` I get:

"Creating images with labels by Pranav.
They are stored in the following folder: C:\Users\lpetrucco\code\DeepLabCut\examples\openfield-Pranav-2018-10-30\labeled-data\m4s1_labeled.
Attention: C:\Users\lpetrucco\code\DeepLabCut\examples\openfield-Pranav-2018-10-30\labeled-data\m4s1 does not appear to have labeled data!
If all the labels are ok, then use the function 'create_training_dataset' to create the training dataset!"

And there is no images in the indicated folder. Ofc, this is not essential for the rest of the notebook to work, and I am able to run the training (but see the other issue). But I don't get the labelled frames. I was able to solve this by replacing in the `check_labels() ` function `os.sep` with `"/"`, as as I said I think the problem is that the path is obtained splitting the relative path in the CollectedData_Pranav.h5, which is separated with "/", with my Windows separator, "\\".

**Comment by MMathisLab** [comment]
Which version of the code are you using?

The latest code does not have any "\" or "/" path separators. 

https://github.com/AlexEMG/DeepLabCut/search?q=os.sep&unscoped_q=os.sep

As we said, this dataset is just meant to be trained on, as it was created in Linux but can be used for test training.

**Comment by vigji** [comment]
I am using version 2.0.6.3, which was installed within the configured anaconda environment. 

Unless we are looking at different branches of the code, in the line that I pointed at in the issue (line 292 of trainingsetmanipulation.py) there is an os.sep which is not compatible with the file that you provide for the demo. the separator is inside the string in the CollectedData_Pranav.h5 file.

It is true that it is minor but could be confusing for Windows users willing to test the installation with the demo

**Comment by MMathisLab** [comment]
Okay yes; the demo was created on Linux, hence why we say just start by loading demo data and running training; it’s just there for you to test your system (beyond just the testscript.py). Thanks for pointing it out.

**Comment by MMathisLab** [comment]
(We will update this in a future release) @meet10may

**Comment by MMathisLab** [comment]
fixed: https://github.com/AlexEMG/DeepLabCut/commit/d8f3482b74d8375e045be940141c6aa0f29e0996
